src/main/java/org/commcare/formplayer/repo/impl/PostgresFormSessionRepo.java

```diff
@@ -58,21 +55,16 @@ public void purgeFormSessions() {
             String deleteQuery = replaceTableName(
                     "delete from %s where custom_safe_cast(dateopened, '2011-01-01'::timestamp) < NOW() - INTERVAL '7 days';"
             );
-            if ("startup".equals(purgeConfig)) {
-                log.info("Beginning state form session purge");
-                long start = System.currentTimeMillis();
-                int deletedRows = this.jdbcTemplate.update(deleteQuery);
-                long elapsed = System.currentTimeMillis() - start;
-                log.info(String.format("Purged %d stale form sessions in %d ms", deletedRows, elapsed));
-            } else {
-                log.debug(String.format("Skipping stale session purge due to current trigger " +
-                        "config '%s'. To enable, set " +
-                        "commcare.formplayer.purge.trigger=startup", purgeConfig));
-            }
+            log.info("Beginning state form session purge");
+            long start = System.currentTimeMillis();
+            deletedRows = this.jdbcTemplate.update(deleteQuery);
+            long elapsed = System.currentTimeMillis() - start;
+            log.info(String.format("Purged %d stale form sessions in %d ms", deletedRows, elapsed));
         } catch (Exception e) {
             // Don't crash for this. Not fatal and prevents start-up
             log.error("Exception purge form sessions", e);
```

Comment:
One last quick point of feedback - This code is no longer on a critical path that would justify swallowing this exception without doing something with it, so it would be good to make sure it ends up in sentry.